runtime/standalone/src/lib.rs

```diff
@@ -56,6 +57,33 @@ pub use impl_versioned_runtime_with_api::*;
 
 pub type CurrencyAdapter = accounts_config::AccountManagerCurrencyAdapter<Balances, ()>;
 
+const DOLLARS: Balance = 1_000_000_000_000;
+const CENTS: Balance = DOLLARS / 100;
```

Comment:
This will need to be smarter since it ties into token decimals. You'll need to move these constants and probably the calculation of these into where token decimals exist.

Reply:
yes, prefer to re-use from primitves/src/monetary.rs

Reply:
fixed, refer to commit [198d16d](https://github.com/t3rn/t3rn/pull/459/commits/198d16dac10fce9162b7ea66037af17646ec448a)

Reply:
fixed, refer to commit [5ebc0bb](https://github.com/t3rn/t3rn/pull/459/commits/5ebc0bbf0953861fd6bb09740e505a68d99a0651)

Reply:
refactored, please refer to commit [b8eea1b](https://github.com/t3rn/t3rn/pull/459/commits/b8eea1b87ad0a84cc8f65f343c8885a26f956f68).